guide/Credential service/Credential Schema.md

```diff
@@ -0,0 +1,140 @@
+# Introduction
+Within the SVX Platform, credential schemas are used to form the basis of a verifiable credential. They structure a set of claims about a Holder and determine the logic behind how the claims are used. Without credential schemas, verifiable credentials would not be able to be issued. 
+
+ 
+# What you need before starting
+The schema file (typically a JSON Schema)
+
+
+# Additional considerations
+Each Credential Schema consists of the following mandatory attributes:
+
+* type (e.g. JsonSchemaValidator2018). The specific type definition determines the content of each data schema.
+
+* id (a URI that identifies the schema file)
+
+Credential schema upload is available to Tenant Administrators and must comply with the following rules:
+
+* A plain JSON object.
+
+* The structure is later checked based on the specification: JSON Schema 
+
+All the attributes in the example are required, but values could be different, e.g. URL address of Schema, name, description etc.
+
+ 
+
+### Example credential schema
+
+```bash
+{ 
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "name": "Example",
+  "description": "Example", 
+  "type": "object", 
+  "properties": { 
+
+    "id": { 
+      "type": "string", 
+    } 
+  }, 
+  "required": ["id"], 
+  "additionalProperties": false, 
+}
+```
+## Create schemas
+**Request**
+
+```bash
+POST/schemas
```

Comment:
missing request header information e.g. authorization & X-Meeco-Organisation-ID 

note X-Meeco-Organisation-ID is not forced yet in dev API, however I think it will be mandatory in future request //cc @linasi

Reply:
Schemas are managed by a tenant admin and later assigned to organisations, so `X-Meeco-Organisation-ID` is not used here.
However, fetching a schema definition that is being assigned to an organisation can be called by the organisation admin and then ` X-Meeco-Organisation-ID` is required.
Let me know if there are any questions.